spec/support/capybara.rb

```diff
@@ -5,8 +5,27 @@
   Capybara.server_port = 9887 + ENV['TEST_ENV_NUMBER'].to_i
 end
 
+Capybara.register_driver :chrome_headless do |app|
+  options = ::Selenium::WebDriver::Chrome::Options.new
+
+  options.add_argument('--headless')
+  options.add_argument('--no-sandbox')
+  options.add_argument('--disable-dev-shm-usage')
+  options.add_argument('--disable-gpu')
+  options.add_argument('--window-size=1400,1400')
+
+  Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
+end
+
+Capybara.javascript_driver = :chrome_headless
+
 RSpec.configure do |config|
   config.before(:each, type: :system) do
     driven_by(:rack_test)
   end
+
+  config.before(:each, type: :system, js: true) do
+    WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
Needed for the chrome headless driver. Do we need to set this back to what it was before the example runs?

Reply:
I tried this in an `around` block that resets back to disabling everything. It looks like something sends a request to clear the session after the end of the around hook (and the `after` one too), which fails if we block requests again after.

Can we just set this in `spec/support/webmock.rb`?